polkadot/xcm/src/lib.rs

```diff
@@ -260,50 +329,68 @@ versioned_type! {
 		V2(v2::Junction),
 		#[codec(index = 3)]
 		V3(v3::Junction),
+		#[codec(index = 4)]
+		V4(v4::Junction),
 	}
 }
 
 versioned_type! {
-	/// A single `MultiLocation` value, together with its version code.
+	/// A single `Location` value, together with its version code.
 	#[derive(Ord, PartialOrd)]
-	pub enum VersionedMultiLocation {
+	pub enum VersionedLocation {
```

Comment:
Did we not agree to also call this `MultiLocation` instead of `VersionedMultiLocation`?

Reply:
I think `VersionedLocation` was good. It's much more understandable